src/twitch.rs

```diff
@@ -105,15 +105,57 @@ pub async fn twitch_irc(mut config: CompleteConfig, tx: Sender<Data>, mut rx: Re
                                 Some(username) => username.to_string(),
                                 None => "Undefined username".to_string(),
                             };
-                            // try to get username from message tags
+                            let mut badges = String::new();
                             if let Some(ref tags) = message.tags {
+                                const VIP_BADGE: char = '\u{1F48E}';
+                                const MODERATOR_BADGE: char = '\u{1F528}';
+                                const SUBSCRIBER_BADGE: char = '\u{2B50}';
+                                const PRIME_GAMING_BADGE: char = '\u{1F451}';
```

Comment:
Constants should be moved to top of file

Reply:
Ok